src/pages/Validators/ValidatorsTable.tsx

```diff
@@ -158,10 +169,14 @@ function ValidatorHeaderCell({
       );
     case "delegator":
       return <GeneralTableHeaderCell header="Delegators" />;
+    case "rewardsEarned":
+      return <GeneralTableHeaderCell header="Rewards Earned" />;
     case "commission":
       return <GeneralTableHeaderCell header="Commission" />;
     case "myDeposit":
```

Comment:
i see your point, these data are not part of "stage 1" in the milestones, but we will eventually need it next week when indexer's ready to have relevant data to serve us? is it okay to keep them in the table? or there's alternative understanding on "not in the mvp"? lmk!

Reply:
I like clean code so I'd say if it's not needed for now let's remove it. We can always refer to the commit history if we want to add it back some day. Because plans change and we don't know for sure we're gonna add it back. If not, we'll end up with unused code and it's hard to maintain over time.

Reply:
that makes sense, its just its a bit difficult to track if we add some of the data not all the data. right now i follow figma as the source of truth for all the info we need to show for delegation. but for the simplicity and clean code, i will remove it from this pr.